tests/rules/pds3/test_ebrocc_xxxx.py

```
@@ -0,0 +1,90 @@
+from tests.rules.support import (
+    PDS3_TEST_RESULTS_DIR as TEST_RESULTS_DIR,
+    associated_abspaths_test,
+    opus_products_test,
+)
+
+
+@pytest.mark.parametrize(
+# Allow duplicated '/volumes/EBROCC_xxxx/EBROCC_0001/BROWSE/ESO1M/ES1_EGB.LBL'
+# and '/volumes/EBROCC_xxxx/EBROCC_0001/BROWSE/ESO1M/ES1_EPB.LBL' here. OPUS
+# will ignore the duplicated items
+    'input_path,expected',
```

Ruff (0.15.21)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)